Mathlib/Algebra/Category/ModuleCat/Adjunctions.lean

```diff
@@ -139,14 +154,17 @@ def μIso (X Y : Type u) :
 
 @[simp]
 lemma μIso_hom_freeMk_tmul_freeMk {X Y : Type u} (x : X) (y : Y) :
-    (μIso R X Y).hom (freeMk x ⊗ₜ freeMk y) = freeMk ⟨x, y⟩ := by
```

Comment:
I'd argue that actually we're missing the lemma that says `(μIso R X Y).hom.hom = (finsuppTensorFinsupp' R _ _).toLinearMap` and similarly for `inv.hom`.

Reply:
@eric-wieser what do you think?